### DISCUSSION TITLE
Motor hold question (Engraving Piano Side) Gravity Problem

### DISCUSSION THREAD

**Discussion by dapeake2** [arkypita/LaserGRBL#1518] [General]
Greetings! 

My Lasermaster2 is tilted 90degrees to engrave the side of my piano (i.e. Y motor vs. gravity). My goal is to use the jog arrows to position the gantry (establishing 0,0) then run the job ( which it does fine on its side). But, due to gravity, the gantry immediately falls after I let go of the jog arrow, making accurate positioning impossible. Is there a way to have the motors hold / clamp? I have tried creating macro buttons that move the axis followed by G4 (dwell) but during G4, the motors release..

Kind thanks for any comments and help!

Dan
Campbell

**Comment by arkypita** [comment]
Yes, via grbl configuration ($1 "step idle delay) you can set number of ms motor stay on after a movement. $1=255 mean infinite.

**Reply by dapeake2** [reply]
That's excellent to hear. I will find that grbl command - thank you so much!!

Peace,

Dan

**Reply by dapeake2** [reply]
To future searchers with this same question: 
 I found what arkypita was referring to in the grbl documentation.

**$1 "step idle delay (ms)"** 

$1=255 is the setting for motors "always enabled".
$1=[1 to 254] is interpreted literally as milliseconds motors are enabled after a step
$1=0 means the motors are disabled as soon as a step is over (in my case with Y vs gravity, Y would drop)

Thanks again!

Dan

**Reply by arkypita** [reply]
thanks, I correct my message
